bundles/automation/org.eclipse.smarthome.automation.integration.test/src/test/groovy/org/eclipse/smarthome/automation/integration/test/AutomationIntegrationJsonTest.groovy

```diff
@@ -260,10 +262,44 @@ class AutomationIntegrationJsonTest extends OSGiTest{
         def action = rule.actions.find{it.id.equals("ItemPostCommandActionID")} as Action
         assertThat action, is(notNullValue())
         assertThat action.typeUID, is("core.ItemCommandAction")
-        assertThat action.configuration.get("itemName"), is("myLampItem")
+        //assertThat action.configuration.get("itemName"), is("myLampItem") -> after bugfix
+        // the only way to check this is to run the rule and check that it is executed successfully
```

Comment:
I don't think we need the comment out line at all (if you add a note in the long description of the commit it should be enough if someone read the history to find the reason why the line has been removed).